Source/ExecutionEngine/CommandLineOptions.cs

```diff
@@ -1206,6 +1206,10 @@ protected override bool ParseOption(string name, CommandLineParseState ps)
           return true;
 
         case "rlimit":
+          ps.GetUnsignedNumericArgument(x => ResourceLimit = Util.BoundedMultiply(x, 1000), null);
+          return true;
+
+        case "reproducibleResourceLimit":
```

Comment:
Can we use a word that captures "reproducible resource that correlates with time", like `tick` or `cycle` ?